From f6e916c3ac836a44140c76ac8819624e53bd0374 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Wed, 22 Jun 2022 16:55:20 -0500
Subject: [PATCH] account: fix build after removal of rest/oauth-proxy.h

librest has removed its OAuth 1 support. This is OK, because we do not
use it. We can just change the header.
---
 gnome-initial-setup/pages/account/gis-account-page-local.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnome-initial-setup/pages/account/gis-account-page-local.c b/gnome-initial-setup/pages/account/gis-account-page-local.c
index c858c608..bca57ee9 100644
--- a/gnome-initial-setup/pages/account/gis-account-page-local.c
+++ b/gnome-initial-setup/pages/account/gis-account-page-local.c
@@ -38,7 +38,7 @@
 #define GOA_API_IS_SUBJECT_TO_CHANGE
 #include <goa/goa.h>
 
-#include <rest/oauth-proxy.h>
+#include <rest/rest-proxy.h>
 #include <json-glib/json-glib.h>
 
 #define VALIDATION_TIMEOUT 600
-- 
GitLab

